src/components/compose/Compose.jsx

```diff
@@ -0,0 +1,21 @@
+import PropTypes from "prop-types";
```

Comment:
i dont understand why you have this component

Reply:
this is a package. it helps to define the types of props.

Reply:
i m not saying about prop types but this compose component

Reply:
oh! sorry.
the `Compose` component, I have used this to compose all the context providers in the `index.js` file of the `src`.
As the providers number was getting increase and the file was getting messy, that's why I created this component.

Reply:
Nice! that thing is new to me, it actually looks clean

Reply:
Yeah, I also search for it when it started to get messy.